hpat/hiframes/series_kernels.py

```diff
@@ -526,7 +526,7 @@ def gt_f(a, b):
     'nlargest_default': lambda A, name: hpat.hiframes.api.init_series(hpat.hiframes.api.nlargest(A, 5, True, gt_f), None, name),
     'nsmallest': lambda A, k, name: hpat.hiframes.api.init_series(hpat.hiframes.api.nlargest(A, k, False, lt_f), None, name),
     'nsmallest_default': lambda A, name: hpat.hiframes.api.init_series(hpat.hiframes.api.nlargest(A, 5, False, lt_f), None, name),
-    'head': lambda A, I, k, name: hpat.hiframes.api.init_series(A[:k], None, name),
+    # 'head': lambda A, I, k, name: hpat.hiframes.api.init_series(A[:k], None, name),
```

Comment:
Is the change exactly needed? I think that affects nothing because you commented out `head` in old style.

Reply:
Yes you right, I recommented it

Reply:
@1e-to In this case tests on parallelism check will be broken. Please refer to any latest PRs, like PR #186

Reply:
@shssf Should I delete existing tests on parallelism and write them in another style, or comment old?

Reply:
@1e-to No, I don't think you need it